client/state/jetpack-settings/settings/test/reducer.js

```diff
@@ -84,9 +84,9 @@ describe( 'reducer', () => {
 					12345678: SETTINGS_FIXTURE[ siteId ]
 				},
 				action = {
-					type: JETPACK_SETTINGS_RECEIVE,
```

Comment:
Looks like there was already a test for _RECEIVE, and the one I updated was a duplicated one. Is it possible ?